### PR TITLE
add basic table

### PR DESCRIPTION
**Pull request by Yimiprod**

### CODE REVIEW

**Comment by Yimiprod** [comment]
@D34THWINGS rebase + base en place

File: gulp/config.js [hunk 1]
```diff
@@ -16,13 +16,13 @@ gulpConfig.vendors = Object.keys(packageJson.dependencies).concat(packageJson.an
 gulpConfig.vendorsDist = `${dist}/js`;
 
 const huntrConfig = {
-  name: 'iTagYou',
+  name: 'HuntR',
```
Comment (D34THWINGS) [inline gulp/config.js]: Spotted xD

Comment (Yimiprod) [inline gulp/config.js]: ;)

**Comment by D34THWINGS** [comment]
```
01 05 2016 18:37:32.601:ERROR [coverage]: PhantomJS 2.1.1 (Linux 0.0.0): Coverage for statements (96.88%) does not meet global threshold (98%)
01 05 2016 18:37:32.603:ERROR [coverage]: PhantomJS 2.1.1 (Linux 0.0.0): Coverage for lines (89.47%) does not meet global threshold (98%)
01 05 2016 18:37:32.603:ERROR [coverage]: PhantomJS 2.1.1 (Linux 0.0.0): Coverage for functions (93.33%) does not meet global threshold (98%)
```

**Comment by Yimiprod** [comment]
Fixed

**Pull request merged**